src/helpers/Utils/usePagination.js

```diff
@@ -24,26 +27,29 @@ const useCreatePageQueryStringKey = (id) => {
  * A pagination helper that tracks the query and resets pagination in case the
  * query changes.
  */
-export const usePagination = (id = null, defaultPage = 1) => {
+export const usePagination = (blockId = null, defaultPage = '1') => {
```

Comment:
I'd think we should stick to a generic `id` rather then `blockId` and a number for the defaultPage.

Reply:
You mean the "blockId" naming or something else?